src/ashley/views.py

```diff
@@ -0,0 +1,117 @@
+"""Views of the ashley django application."""
+
+import logging
+from abc import ABC, abstractmethod
+from typing import List
+
+from django.contrib.auth import authenticate, login
+from django.http import (
+    HttpResponse,
+    HttpResponseBadRequest,
+    HttpResponseForbidden,
+    HttpResponseRedirect,
+)
+from django.urls import reverse
+from django.utils import translation
+from django.utils.decorators import method_decorator
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+from machina.apps.forum_permission.shortcuts import assign_perm
+from machina.core.db.models import get_model
+
+from lti_provider.exceptions import LTIException
+from lti_provider.lti import LTI
+
+from .settings import settings
+
+Forum = get_model("forum", "Forum")  # pylint: disable=C0103
+LTIContext = get_model("ashley", "LTIContext")  # pylint: disable=C0103
+
+logger = logging.getLogger("ashley")
+
+
+@method_decorator(csrf_exempt, name="dispatch")
+class BaseLTIView(ABC, View):
+    """Forum view called by an LTI launch request."""
+
+    def post(self, request, *args, **kwargs) -> HttpResponse:  # pylint: disable=W0613
+        """Handler for POST requests."""
+        lti_request = LTI(request)
+        try:
+            lti_request.verify()
+            user = authenticate(request, lti_request=lti_request)
+            if user is not None:
+                login(request, user)
+                return self._do_on_login(lti_request)
+            return HttpResponseForbidden("Forbidden")
```

Comment:
If you have nothing else to say, I think you can just :
```suggestion
            return HttpResponseForbidden()
```